### PR TITLE
Stub MORE css classnames in storybook snapshot tests

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What
* Overriding getClassName was running in the beforeEach and not at global scope.
* Adding resetIds also (typically its just the classnames that are causing issues, we can monitor this one but it doesn't have a good way to override it)

# Why
Previous PR #1963 completed a little early - found some more when looking at the snapshots in the PR.

### CODE REVIEW

File: packages/storybook/stories/storybook.test.ts [hunk 1]
```diff
@@ -15,15 +15,18 @@ jest.mock('@azure/communication-calling', () => {
 
 ReactDom.createPortal = (node: any) => node;
 
-// Classnames in fluent are of the format css-#, where # is an integer that is incremented by a global singleton
+// Classnames in fluent are of the format <display name>-#, where # is an integer that is incremented by a global singleton
```
Comment (mgamis-msft) [inline packages/storybook/stories/storybook.test.ts:18]: Sorry, I don't understand this. The display name of what?

Comment (JamesBurnside) [reply packages/storybook/stories/storybook.test.ts:18]: Just what fluent calls it: https://github.com/microsoft/fluentui/blob/66dc6e9684baf35e7930622e1f3accdb51ba8516/packages/merge-styles/src/Stylesheet.ts#L44
I'll update this to `<classname-prefix>-#` and add an example for clarity!

Comment (JamesBurnside) [reply packages/storybook/stories/storybook.test.ts]: updated

**Comment by JamesBurnside** [comment]
Still seeing some but very few:
image

Lets check this in and see how much these still existing ones impact PRs

**Review by mgamis-msft — APPROVED** [review]

**Review by prprabhu-ms — APPROVED** [review]

**Pull request merged**